agent/consul/replication_test.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 func TestReplicationRestart(t *testing.T) {
-	mgr := NewLeaderRoutineManager(testutil.TestLogger(t))
```

Comment:
Repeat of the comment that this would regress and cease sending logs down through the `*testing.T.Logf` function.